### PR TITLE
[TF Led] Fix flaky TF Led test

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

The reason why the TF LED test is flaky was not fully fixed in: https://github.com/huggingface/transformers/pull/9459
and is actually the following: 

Currently the `decoder_attention_mask` can have a `0` at its first input:

```python
decoder_attention_mask[:, 0] == 0
```

Since the decoder uses a causal mask, this however leads to problems as a softmax over only very large negative numbers in computed. Now since TF and PT use slightly different large numbers, we can see significant differences between the models. The solution is to make sure that the `decoder_attention_mask` used for the `tf_pt_equivalence` test cannot be zero at the first position (I've done the same changes for all TFBart models in: https://github.com/huggingface/transformers/pull/9497 and also made sure in https://github.com/huggingface/transformers/pull/9497 that the TF templates are correctly updated )

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
cc @LysandreJik @sgugger @jplu

**Comment by sgugger** [comment]
Thanks for fixing!
